### DISCUSSION TITLE
Convert CreateAction Slideover to an EditAction Slideover After Record Created?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#14445] [Help]
### Package

Actions

### Package Version

v3

### How can we help you?

I'm displaying a relation manager's table's CreateAction in a slideover modal and would like to update the slideover with the  new record data as an EditAction so don't have to redirect user.   Maybe by using the `Tables\Actions\CreateAction` `after()` method?  Is this possible?

**Comment by leandrocfe** [comment]
Could you provide more info?

**Reply by myregistration** [reply]
Basically, I just want to refresh the Create Record slideover modal with the new record's data and display the Edit Record UI instead of redirecting the user to Edit page.
